tests/e2e/specs/editor/media/insertWebMVideo.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { percySnapshot } from '@percy/puppeteer';
+
+/**
+ * Internal dependencies
+ */
+import { createNewStory, previewStory, insertStoryTitle } from '../../../utils';
+
+const MODAL = '.media-modal';
+
+describe('Inserting WebM Video', () => {
+  it('should insert an video by clicking on media dialog it', async () => {
+    await createNewStory();
+
+    await expect(page).not.toMatchElement('[data-testid="FrameElement"]');
+
+    // Clicking will only act on the first element.
+    await expect(page).toClick('button', { text: 'Upload' });
+
+    await page.waitForSelector(MODAL, {
+      visible: true,
+    });
+    const btnTab = '#menu-item-browse';
+    await page.waitForSelector(btnTab);
+    await page.evaluate((selector) => {
+      document.querySelector(selector).click();
+    }, btnTab);
```

Comment:
This whole part is also used in other tests - making it a util would increase readability a lot

Reply:
Sorry, I should have been more clear. I meant a util for `insertMediaFromDialog()` or something, not a `clickButton` util.

Especially since

```
await page.waitForSelector(selector);
  await page.evaluate((btn) => {
    document.querySelector(btn).click();
  }, selector);
```

can just be replaced by `await expect(page).toClick('button', { text: 'Home' })` (`text` is optional)

Clicking on buttons by type & label should be preferred over some selectors.

Reply:
Why is clicking by text better? CSS selector seem like a more solid way of doing this? 

Do you want me to change this?

Reply:
Also think some of these refactors are out of the scope of a 1 point...

Reply:
Can also create a new ticket for refactoring other tests. Main thing is that it gets done :)

Clicking by text and roles is a best practice as it does better resemble how the software is used. CSS selectors can always change (e.g. due to refactors) and should not be relied upon for these actions.

Just like how we avoid `data-testid` in other tests whenever possible.